src/render/quantum.rs

```diff
@@ -135,10 +138,10 @@ impl AudioRenderQuantumChannel {
 use std::ops::{Deref, DerefMut};
 
 impl Deref for AudioRenderQuantumChannel {
```

Comment:
This here is the motivation of the PR. We stop hardcoding the length of the render quantum to 128, but leave it unspecified (we return a [slice](https://doc.rust-lang.org/std/primitive.slice.html), instead of an [array](https://doc.rust-lang.org/std/primitive.array.html)).
This incurs a small performance hit (a percent, looking at the benchmarks) because more bound checks need to be performed at runtime.
However, this opens the way for the "dynamic render quantum size"  feature without breaking the current API. With this PR merged, the `RENDER_QUANTUM_SIZE` (or, the value 128) is no longer present in the API surface